libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -130,6 +130,13 @@ const AP_Param::GroupInfo AP_Follow::var_info[] = {
     AP_GROUPINFO("_ALT_TYPE", 10, AP_Follow, _alt_type, AP_FOLLOW_ALT_TYPE_DEFAULT),
 #endif
 
+    // @Param: _OPTIONS
+    // @DisplayName: Follow options
+    // @Description: Follow options bitmask. Specifies mount behaviour in follow mode
+    // @Values: 0:None,1: Mount Follows lead vehicle
+    // @User: Standard
+    AP_GROUPINFO("_OPTIONS", 11, AP_Follow, _options, 0),
```

Comment:
Can't use index 0 for parameters.  The maths doesn't work out in our fancy parameter storage system.

Reply:
@peterbarker I think this is using index 11 so it's OK.  The default is zero..

Reply:
Yes, sorry, I completely misread the line